File: benchmarks/poly/loopsum/loopsum.js
function main() {
  const _t0 = process.hrtime.bigint();
  let acc = 0;
  for (let i = 1; i <= 10000000; i++) {
    acc = (acc + i * i) % 1000000007;
  }
  console.log(acc);
  console.error(`bench_ns=${process.hrtime.bigint() - _t0}`);
}

main();

File: benchmarks/poly/loopsum/loopsum.rs
fn main() {
    let t0 = std::time::Instant::now();
    let mut acc: i64 = 0;
    for i in 1..=10_000_000i64 {
        acc = (acc + i * i) % 1_000_000_007;
    }
    println!("{}", acc);
    eprintln!("bench_ns={}", t0.elapsed().as_nanos());
}
